templates/citizen_project/_form.html.twig

```diff
@@ -69,33 +85,43 @@
 
 <div class="form__new" id="field-problem-description">
     {{ form_label(form.problem_description, 'À quelle problématique locale répond votre projet ?', {label_attr: {class: 'label__header--large'}}) }}
-    {{ form_errors(form.problem_description) }}
-    {{ form_widget(form.problem_description, {
-        'attr': {
-            'class': 'form--full form__field--textarea-120',
-            'placeholder': 'Décrivez en 1 ou 2 phrases le problème local auquel vous proposez de répondre.',
-        }
-    }) }}
-    <small>
-        Ex: "Les collégiens doivent effectuer un stage d'observation en classe de 3e ou de 4e
-        pour les aider à déterminer leur orientation professionnelle mais
-        ils ont parfois du mal à le trouver en dehors de leur réseau famillial."
-    </small>
+    {% if form.problem_description.vars.disabled %}
+        <div id="citizen_project_problem_description_text">{{ form.vars.data.problemDescription }}</div>
+        <div class="visually-hidden">{{ form_widget(form.problem_description) }}</div>
+    {% else %}
+        {{ form_errors(form.problem_description) }}
+        {{ form_widget(form.problem_description, {
+            'attr': {
+                'class': 'form--full form__field--textarea-120',
+                'placeholder': 'Décrivez en 1 ou 2 phrases le problème local auquel vous proposez de répondre.',
+            }
+        }) }}
+        <small>
+            Ex: "Les collégiens doivent effectuer un stage d'observation en classe de 3e ou de 4e
+            pour les aider à déterminer leur orientation professionnelle mais
+            ils ont parfois du mal à le trouver en dehors de leur réseau famillial."
+        </small>
+    {% endif %}
 </div>
 
 <div class="form__new" id="field-proposed-solution">
     {{ form_label(form.proposed_solution, 'Comment votre projet tend à répondre à ce problème ?', {label_attr: {class:'label__header--large'}}) }}
-    {{ form_errors(form.proposed_solution) }}
-    {{ form_widget(form.proposed_solution, {
-        'attr': {
-            'class': 'form--full form__field--textarea-120',
-            'placeholder': 'Un court paragraphe sur la ou les solutions que vous proposez',
-        }
-    }) }}
-    <small>
-        Ex: Le projet a pour objectif de mettre en relation ces élèves avec un réseau de professionnels
-        (entreprises, commerçants, artisans, etc.) volontaires pour les accueillir.
-    </small>
+    {% if form.proposed_solution.vars.disabled %}
+        <div id="citizen_project_proposed_solution_text">{{ form.vars.data.proposedSolution|raw }}</div>
```

Comment:
why `|raw` filter?

Reply:
Yes, because this field contains the HTML tags. Its value was entered in the CKEditor and we need to display a formatted text.